source/deployment/resyncing-conda.rst

```diff
@@ -10,19 +10,18 @@ mid-cycle due to some unintended upgrade.
 Before the start of each cycle, we re-synchronize our internal collection of
 packages with the latest packages available from Anaconda.
 
-The internal Anaconda server runs on the ``pergamon`` host, inside the Controls
-netowork, and it runs as a service account user ``anacodna-server``.
+The internal Anaconda server runs on the ``alexandria`` host, inside the Controls
+network, and it runs as a service account user ``anaconda-server``.
 
 .. code-block:: bash
 
-ssh pegramon
-sudo su anaconda-server
-cd
-export https_proxy=http://proxy:8888
-export HTTPS_PROXY=http://proxy:8888
+    ssh alexandria
+    sudo su anaconda-server cd export
```

Comment:
First of all, something happened with the lines here, which made it wrong instructions. Second, I don't think we use the anaconda-server user on the machine. @licode, can you confirm?

Reply:
Thanks @mrakitin, these few lines where not part of the code-block which is why I attempted to fix them. It appears that I didn't arrange them correctly however, I have made the change.